language-design.md

```diff
@@ -38,7 +55,38 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+Language design is tricky, and it's hard to get it right. Determining which
+languages are good is important to know which ones to use --- even a DSL that is
+designed exactly for your purpose is not a good choice if it is poorly designed.
+Steele would say, and we would agree, that a good general purpose language is
+not necessarily marked by how good its syntax is or how many features it has but
+rather by its ability to grow.  No language can cover all use cases, and if
```

Comment:
It's true that a language can't cover every use case. Ideally, we should be selecting GPLs for their use rather than popularity.